### PR TITLE
feat: dim points on tag selection

### PR DESCRIPTION
**Pull request by raphael-arce**

### CODE REVIEW

**Review by aeschi — APPROVED** [review]
LGTM! 👍

File: src/components/tags.tsx [hunk 1]
```diff
@@ -5,10 +5,10 @@ export function Tags() {
 	const tags = new Set(contacts.map((contact) => contact.tags).flat());
 
 	return (
-		<div className="flex justify-center pt-20">
+		<div className="flex justify-center pt-10">
 			<div className="flex flex-col items-center gap-y-2">
 				<div className="text-lg">Filtere mit Tags</div>
-				<div className="flex gap-x-3 w-[31rem] flex-wrap gap-y-3 justify-center">
+				<div className="flex gap-x-3 w-[37rem] flex-wrap gap-y-3 justify-center">
```
Comment (aeschi) [inline src/components/tags.tsx]: I think the width can be even wider and more responsive like: w-10/12

**Review by aeschi — APPROVED** [review]
LGTM!

since you made the size of XIcon adaptable - it should be 
size-8 on the Splash screen.

File: src/components/tags.tsx (hunk 1 above)
Comment (raphael-arce) [reply src/components/tags.tsx]: okay, will change 👍

**Comment by raphael-arce** [comment]
@aeschi my bad, I fixed it! could you have another look? thx! :)

**Comment by aeschi** [comment]
Looks good! 👍

**Pull request merged**